.github/workflows/system-tests.yaml

```diff
@@ -10,6 +10,6 @@ jobs:
     uses: precice/tutorials/.github/workflows/run_testsuite_workflow.yml@develop
     with:
       suites: release_test
-      build_args: PRECICE_REF:${{ github.event.pull_request.head.sha }}
+      build_args: PRECICE_REF:${{ github.event.pull_request.head.sha }},OPENFOAM_EXECUTABLE:openfoam2312,OPENFOAM_ADAPTER_REF:master,TUTORIALS_REF:master,PLATFORM:ubuntu_2204,PYTHON_BINDINGS_REF:master,FENICS_ADAPTER_REF:master,CALULIX_VERSION:2.20,CALULIX_ADAPTER_REF:master,SU2_VERSION:7.5.1,SU2_ADAPTER_REF:master
```

Comment:
specifying `master` (or any other branch) will eventually lead to false-positive cache hits. Fine for a draft, but we should not merge it like this.

Reply:
Replaced the `master` with a `gather-refs` for each repository. This actually targets the develop branch for now, we can decide later on the exact strategy and add more workflows/combinations.